snekbox/api/resources/eval.py

```diff
@@ -122,10 +124,21 @@ def on_post(self, req: falcon.Request, resp: falcon.Response) -> None:
         if "input" in body:
             body.setdefault("args", ["-c"])
             body["args"].append(body["input"])
+
+        binary_path = body.get("binary_path")
+        if binary_path:
+            binary_path = Path(binary_path)
+            if (
+                not binary_path.resolve().as_posix().startswith("/lang/")
```

Comment:
Should we move the `resolve().as_posix()` calls above so the pathlib calls below can make use of it? Or do they mutate the internal object (source of eldritch horrors)

Reply:
changed this func quite a bit since then, so this is no longer relevant